core/components/blocks/Checkout/OrderReview.js

```diff
@@ -11,11 +11,17 @@ export default {
   data () {
     return {
       isFilled: false,
+      placeOrderVisible: true,
       orderReview: {
         terms: false
       }
     }
   },
+  created () {
+    this.$bus.$on('place-order-button-visible', (val) => {
```

Comment:
Please rename it to: checkout-before-placeOrder

Reply:
No, it event for separate goal.

Reply:
@pkarw paypal integration example https://github.com/develodesign/vsf-payment-paypal/pull/41/files

Reply:
It solve integration with PayPal and for other payment methods it works out of the box.
If we figure out how to improve - it will be cool.